packages/synapse-react-client/src/components/EntityDownloadButton/EntityDownloadButton.tsx

```diff
@@ -265,6 +294,26 @@ export function EntityDownloadButton(props: {
     props.entityType,
   )
 
+  // Get context and download functionality
+  const { downloadCartPageUrl } = useSynapseContext()
+  const { mutate: addFileToDownloadList } = useAddFileToDownloadList({
+    onSuccess: () => {
+      displayFilesWereAddedToDownloadListSuccess(downloadCartPageUrl)
```

Comment:
FIXME: This message displays to the user that files have been successfully added to their download cart even when there are zero files in a folder.